drizzlepac/haputils/make_poller_files.py

```diff
@@ -117,35 +133,39 @@ def locate_fitsfile(search_string):
          full file path + image name of specified search_string.
     """
     if search_string.endswith("_flt.fits") or search_string.endswith("_flc.fits"):  # Process search_string as a full filename
-        # Look for files in CWD first
-        if os.path.exists(search_string):
+        # Look in user-provided path (assuming they provided one)
+        if os.path.exists(search_string) and "/" in search_string:
+            return search_string
+        # Look for files in CWD
+        if os.path.exists(search_string) and "/" not in search_string:
             return os.getcwd()+"/"+search_string
         # If not found in CWD, look elsewhere...
         if not os.getenv("DATA_PATH"):
             sys.exit("ERROR: Undefined online cache data root path. Please set environment variable 'DATA_PATH'")
-        fullfilepath = "{}/{}/{}/{}".format(os.getenv("DATA_PATH"), search_string[:4], search_string, search_string)
+        fullfilepath = "{}/{}/{}/{}".format(os.getenv("DATA_PATH"), search_string[:4], search_string[:-9], search_string)
```

Comment:
Again, a more platform-independent way to generate filenames would be to use 'os.sep'..

Reply:
issue resolved by commit [6626a45](https://github.com/spacetelescope/drizzlepac/pull/1138/commits/6626a452591a0f7331157f0954be141be06cfa4f)